s4e-web/src/app/views/settings/profile/profile.component.ts

```diff
@@ -1,6 +1,8 @@
+import { ProfileService } from './../../../state/profile/profile.service';
 import {Component, OnInit} from '@angular/core';
 import {SessionQuery} from '../../../state/session/session.query';
 import {Observable} from 'rxjs';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
```

Comment:
In componens / forms we generally use `@ng-stack/forms` instead of `@angular/forms`. They work the same, but provide strong typings.